v2/concordium/types.proto

```diff
@@ -1323,6 +1323,19 @@ message TimeoutParameters {
   Ratio timeout_decrease = 3;
 }
 
+// Finalization committee parameters used from protocol version 6
+message FinalizationCommitteeParameters {
+  // The minimum number of bakers that can take part
+  // of the finalization committee.
```

Comment:
This phrasing is confusing. It is odd to say "minimum" that "can".

If I read this I wonder whether this is a lower bound or something else. And it is not a lower bound in the sense that there can be fewer finalizers. Please elaborate.

Reply:
// The minimum size of a finalization committee